packages/slider/slider-handle.md

```diff
@@ -1,7 +1,12 @@
 ## Description
 
 Some advanced slider uses require more than one handle. One example of this is the
-range slider above. `sp-slider` supports an arbitrary number of handles via the `<sp-slider-handle>` sub-component, although it would be very rare to ever require more than two handles.
+range slider above. `sp-slider` supports multiple handles via the `<sp-slider-handle>` sub-component, although it would be very rare to ever require more than two handles.
```

Comment:
```suggestion
range slider above. `<sp-slider>` supports multiple handles via the `<sp-slider-handle>` sub-component, although it would be very rare to ever require more than two handles.
```